src/__tests__/element-queries.js

```diff
@@ -1092,6 +1102,19 @@ test('the default value for `ignore` can be configured', () => {
   expect(noStyle[1].tagName).toBe('DIV')
 })
 
+test('the default value for `ignore` is used in errors', () => {
```

Comment:
New test that proves that overriding `defaultIgnore` produces a new error message